scripts/vagrant/multi/README.md

```diff
@@ -0,0 +1,76 @@
+# Vagrant
+
+This allows you to run a Kubernetes environment on multiple boxes using vagrant to provision VMs either locally or in a cloud environment.
+Let's us compare a feature branch to release by specifying a `FEATURE_DOCKER_IMAGE` env variable.
+
+It includes:
+- kubernetes (using kind)
+- etcd (single node)
+- m3db operator
+- m3db node (single node)
+- m3coordinator dedicated (two instances)
+- prometheus
+- grafana (accessible localhost:3333, login admin:admin)
+
+This is useful for benchmarking and similar needs.
+
+# Requirements
+Setup vagrant azure provider via [docs](https://github.com/Azure/vagrant-azure).
```

Comment:
nit: should we add this in the single node setup too?

Reply:
There's no azure setup added for single node.